charts/spire/charts/spire-agent/templates/daemonset.yaml

```diff
@@ -43,6 +43,22 @@ spec:
           args: ["-t", "30", "-h", "{{ include "spire-agent.server-address" . | trim }}", "-p", {{ .Values.server.port | quote }}]
           resources:
             {{- toYaml .Values.waitForIt.resources | nindent 12 }}
+        {{- if gt (dig "fsGroup" 0.0 .Values.podSecurityContext) 0.0 }}
```

Comment:
Unfortunately, helm is casting integers to floats. So when I added
```yaml
podSecurityContext:
  fsGroup: 0
```

it was failing the gt test because float != int with
gt .Values.podSecurityPolicy.fsGroup 0.
Casting all the compared 0's in the helm template to 0.0 fixed the test.

Would a comment like:
{{/* casting 0's to floats via 0.0 to fix gt comparison */}}
The line above help future readers of the code?

Reply:
I found a different way to fix it. can cast it back to an int. Will change it to do that.

Reply:
Done